modules/appeals_api/config/schemas/v2/200996_headers.json

```diff
@@ -47,6 +47,31 @@
           { "description": "Veteran's insurance policy number", "maxLength": 18 },
           { "$ref": "#/definitions/nonBlankString" }
         ] },
+        "X-Non-VA-Claimant-SSN": {
```

Comment:
Let's remove `Non-VA` and just make all these `X-Claimant-*`

Reply:
That's how I began the change, but moved for more clarity. Maybe being too nit-picky, but technically the Veteran filling it out on their own behalf is the claimant. Thinking of external users of our API, or should we assume they should understand the difference?

Reply:
Drew and I discussed outside of this thread. Will reduce the key length and add comments if necessary and make crystal clear in documentation if not already clear.